compiler-plugin-test/src/test/kotlin/it/unibo/collektive/test/TestLoopWithoutAlignWarning.kt

```diff
@@ -0,0 +1,62 @@
+package it.unibo.collektive.test
+
+import io.kotest.core.spec.style.FreeSpec
+import it.unibo.collektive.test.util.CompileUtils
+import it.unibo.collektive.test.util.CompileUtils.ProgramTemplates
+import it.unibo.collektive.test.util.CompileUtils.noWarning
+import it.unibo.collektive.test.util.CompileUtils.warning
+import org.jetbrains.kotlin.compiler.plugin.ExperimentalCompilerApi
+
+@OptIn(ExperimentalCompilerApi::class)
+class TestLoopWithoutAlignWarning : FreeSpec({
+    "A single aggregate function called inside a loop: " - {
+        val testingProgramTemplate = CompileUtils.testingProgramFromTemplate(
+            ProgramTemplates.SINGLE_AGGREGATE_IN_A_LOOP,
+        )
+        listOf(
+            "exampleAggregate" to "exampleAggregate()",
+            "neighboring" to "neighboring(0)",
+        ).forEach {
```

Comment:
I prefer tuple-unpacking for more clear code.

```kotlin
.forEach { (functionName, functionCall) -> .... }
```

Reply:
it's called "destructuring declaration" in Kotlin jargon, but I agree.